src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -5,15 +5,25 @@ import { FORMS as RemediationForms } from '../../ion/RemediationConstants';
 const Body = BaseForm.extend({
   title() {
     if (this.options.appState.getCurrentViewState().href.endsWith('idp/idx/enroll/update')) {
-      return loc('oie.primaryauth.submit', 'login');
+      const attributes = this.options.appState.attributes;
```

Comment:
can we also check for `this.options.appState?.attributes` just in case..

Reply:
Good suggestion! I will do that in the text localization ticket for progressive profiling as I just did the merge. Thank you!

Reply:
I just realized that if the logic meets `if this.options.appState.` condition in line 7 which means `this.options.appState` will not be null?

Reply:
`attributes` could be undefined right ?

Reply:
Correct me if I am wrong `this.options.appState?.attributes` This is to check `appState` valid or not?
if `attributes` is undefined, it will be caught by `attributes?.uiDisplay?.label` ?